ipstack/help.md

```diff
@@ -17,9 +17,20 @@
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|cred_token|credential_token|None|True|API Token|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|cred_token|credential_secret_key|None|True|API Token|None|349037a927fc2abfd850349c3acd95cdb|
```

Comment:
Can we keep the examples between raw JSON section and table the same? We will autogenerate them from the examples in the future so when that happen the raw JSON and markdown tables will be the same.

Reply:
Same for action inputs